### PR TITLE
[python coverage]fix result color bar

### PR DESCRIPTION
**Pull request by pettershao-ragilenetworks**
What I did
fix python gcov warning "Please install dotnet core to enable automatic generation of Html report"
more info check: Azure/sonic-snmpagent#215
How I did it
install dotnet core
How to verify it

### CODE REVIEW

**Comment by pettershao-ragilenetworks** [comment]
@xumia @lguohan hi , help forward, thanks!

File: azure-pipelines.yml [hunk 1]
```diff
@@ -92,6 +92,16 @@ jobs:
         workingDirectory: $(Pipeline.Workspace)/target/python-wheels/
         displayName: 'Install Python dependencies'
 
+      - script: |
+          set -ex
+          # Install .NET CORE
+          wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+          sudo dpkg -i packages-microsoft-prod.deb
```
Comment (jleveque) [inline azure-pipelines.yml:99]: Why download this package? Can you not simply use `apt-add-repository` as demonstrated in the "Ubuntu 20.04 (Focal)" section here: https://docs.microsoft.com/en-us/windows-server/administration/linux-package-repository-for-microsoft-software.

**Review by jleveque — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml [hunk 2]
```diff
@@ -92,6 +92,16 @@ jobs:
         workingDirectory: $(Pipeline.Workspace)/target/python-wheels/
         displayName: 'Install Python dependencies'
 
+      - script: |
+          set -ex
+          # Install .NET CORE
+          curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+          sudo apt-add-repository https://packages.microsoft.com/debian/10/prod
+          sudo apt-get update
+          sudo apt-get -y install -f
```
Comment (lguohan) [inline azure-pipelines.yml:101]: do we still need this command?

Comment (pettershao-ragilenetworks) [reply azure-pipelines.yml]: fixed, help forward!

**Review by lguohan — APPROVED** [review]

**Pull request merged**